sdk/maps/Azure.Maps.Search/README.md

```diff
@@ -0,0 +1,185 @@
+    Language = "en"
+});
+```
+
+```C# Snippet:SearchStructuredAddress
+var client = CreateClient();
+var address = new StructuredAddress {
+    CountryCode = "US",
+    StreetNumber = "15127",
+    StreetName = "NE 24th Street",
+    Municipality = "Redmond",
+    CountrySubdivision = "WA",
+    PostalCode = "98052"
+};
+var searchResult = await client.SearchStructuredAddressAsync(address);
+```
+
+```C# Snippet:SearchInsideGeometry
+var client = CreateClient();
+
+var sfPolygon = new GeoPolygon(new[]
+{
+    new GeoPosition(-122.43576049804686, 37.752415234354402),
+    new GeoPosition(-122.4330139160, 37.706604725423119),
+    new GeoPosition(-122.36434936523438, 37.712059855877314),
+    new GeoPosition(-122.43576049804686, 37.7524152343544)
+});
+
+var taipeiPolygon = new GeoPolygon(new[]
+{
+    new GeoPosition(121.56, 25.04),
+    new GeoPosition(121.565, 25.04),
+    new GeoPosition(121.565, 25.045),
+    new GeoPosition(121.56, 25.045),
+    new GeoPosition(121.56, 25.04)
+});
+
+var searchResponse = await client.SearchInsideGeometryAsync("coffee", new GeoCollection(new[] { sfPolygon, taipeiPolygon }), new SearchInsideGeometryOptions {
+    Language = "en"
+});
+```
+
+```C# Snippet:SearchAddress
+var client = CreateClient();
+var searchResult = await client.SearchAddressAsync("Seattle");
+```
+
+## Troubleshooting
+
+### General
+
+When you interact with the Azure Maps Services, errors returned by the Language service correspond to the same HTTP status codes returned for REST API requests.
+
+For example, if you try to search with invalid coordinates, a error is returned, indicating "Bad Request".400
+
+## Next steps
+
+* [More detailed samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Search/tests/Samples)
+
+## Contributing
+
+See the [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) for details on building, testing, and contributing to this library.
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit <cla.microsoft.com>.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact <opencode@microsoft.com> with any additional questions or comments.
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net/sdk/template/Azure.Template/README.png)
```

Comment:
Update the link to: https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net/sdk/maps/Azure.Maps.Search/README.png